samples/stock/plotoptions/series-datagrouping-enabled/demo.js

```diff
@@ -0,0 +1,22 @@
+const data = Array.from({ length: 800 }, (_, i) =>
+    Math.round((2 * Math.sin(i / 50) + Math.random() - 0.5) * 100)
+);
+
+Highcharts.stockChart('container', {
+    legend: {
+        enabled: true
+    },
+    series: [
+        {
+            data: data,
+            name: 'Data Grouping enabled',
+            dataGrouping: {
+                enabled: false
+            }
+        },
+        {
+            name: 'Data Grouping disabled',
+            data
```

Comment:
I think the DG here should be forced. Or increase the number of points. On a wide screen, I had two exactly the same series rendered (both without DG)